### PR TITLE
Remove `stream` option from `RawKernel` and add missing docs of arguments in `ReductionKernel`

### PR DESCRIPTION
**Pull request by kmaehashi**

### CODE REVIEW

File: cupy/core/elementwise.pxi [hunk 1]
```diff
@@ -541,6 +541,7 @@ cdef class ElementwiseKernel:
                 This parameter must be specified if and only if all ndarrays
                 are `raw` and the range size cannot be determined
                 automatically.
+            stream (cupy.cuda.Stream): CUDA stream to run this kernel.
```
Comment (okuta) [inline cupy/core/elementwise.pxi]: We can use current stream. This option should be deprecated IMO.

File: cupy/core/reduction.pxi [hunk 2]
```diff
@@ -360,6 +360,11 @@ class ReductionKernel(object):
 
         Args:
             args: Arguments of the kernel.
+            axis (int or tuple of ints): Axis or axes along which the
+                reduction is performed.
+            keepdims (bool): If ``True``, the specified axes are remained as
+                axes of length one.
+            stream (cupy.cuda.Stream): CUDA stream to run this kernel.
```
Comment (okuta) [inline cupy/core/reduction.pxi]: ditto.

**Comment by kmaehashi** [comment]
Fixed. I also removed `stream` option from `RawKernel`.

**Comment by okuta** [comment]
OK, LGTM!

**Comment by okuta** [comment]
jenkins, test this please.

**Pull request merged**